.travis.yml

```diff
@@ -31,3 +36,6 @@ deploy:
   on:
     tags: true
     go: 1.8
+
+after_deploy:
+  - make docker-push
```

Comment:
There's no docker-push target in https://github.com/bblfsh/dashboard/blob/master/Makefile